src/client/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import './footer.css'
+
+export const Footer = () => {
+
+    return <footer>
+        <div className='footer'>
+            <div className="column">
+                <h3>Company</h3>
+                <ul>
+                    <li>About us</li>
```

Comment:
@dpfernandes do we need the links here or is that not part of the project's scope?

Reply:
And the third column (Sign up), with the email input field, is that needed? @dpfernandes

Reply:
@valmdz and @semih1239 we are going to leave the Signup out of the scope